backend/app/views/coupons/_list.json.jbuilder

```diff
@@ -2,22 +2,6 @@
 
 json.partial! 'coupons/base', coupon: coupon
 
-if coupon.campaign_charity.present?
-  json.charity do
-    json.partial! 'charities/minimal', charity: coupon.campaign_charity.charity
-  end
-else
-  json.charity nil
-end
-
-if coupon.secondary_donation.present?
-  json.secondaryDonation do
-    json.partial! 'secondary_donations/secondary_donation', secondary_donation: coupon.secondary_donation
-  end
-else
-  json.secondaryDonation nil
-end
-
```

Comment:
Removed in https://github.com/Giving-Coupons/giving-coupons/pull/373 
<img width="561" alt="image" src="https://user-images.githubusercontent.com/75519136/200111968-4eeb48cc-9724-4228-8d41-99e5d882178a.png">

but forgot to remove the backend view.